### DISCUSSION TITLE
For you

### DISCUSSION THREAD

**Discussion by bundubuduri** [community/community#16677] [Other Feature Feedback, Questions, & Ideas]
Tab to toggle between:

-  for you repos  
- for you people

**Comment by the-lay** [comment]
Moving releases into a separate tab might also be a nice idea.

**Reply by bundubuduri** [reply]
Agreed. Im sure its somewhere within the backlog/pipeline.

**Reply by dewedeth** [reply]
thanks

**Comment by ripred** [comment]
Came in just to say I was a fan of the "For You" tab.  I get that it's not everybody's cup of tea but you don't have to look at it.  And for all of the big data gathering that Mirosoft's gonna do on the activity anyway I might as well take a look at its best guess at stuff I'd find relevant

**Reply by bundubuduri** [reply]
@ripred Yea I think this feature is a pilot and we will definitely see more iterations in the future. I actually like it as I have discovered interesting people and projects. But most things that pop up are not relevant so, I have to really dig and I don't really want to do that.

**Comment by MightyPork** [comment]
One thing I wish was in the tab is stuff I recently interacted with, my last issues, contributed to repos etc. - things I may want to check on later. Currently it's quite difficult to find.

The rest of the card, and "following", is pretty much useless for me. I don't use Github as a social media platform, but to contribute to open source projects.

**Reply by bundubuduri** [reply]
That is  a good one. I actually like the social aspect of it. I also use Bitbucket and its not as social so there is a balance between the two. Most developers that I come across are more active on Github. This is also the case for opensource projects.

**Reply by Oskarsky-1979** [reply]
I'm trying

**Comment by Grek1313** [comment]
> Tab to toggle between:
> 
> * for you repos
> * for you people
